utils/constants.ts

```diff
@@ -3,7 +3,7 @@ import { ether } from '../utils/units';
 
 export const AUCTION_TIME_INCREMENT = new BigNumber(30); // Unix seconds
 export const DEFAULT_AUCTION_PRICE_NUMERATOR = new BigNumber(1374);
-export const DEFAULT_AUCTION_PRICE_DENOMINATOR = new BigNumber(1000);
+export const DEFAULT_AUCTION_PRICE_DIVISOR = new BigNumber(1000);
```

Comment:
These should eventually be funned down to set-protocol-utils. Not sure when we stopped doing this to iterate faster on contracts